src/Microsoft.VisualStudio.Web.CodeGeneration.Core/Microsoft.VisualStudio.Web.CodeGeneration.Core.csproj

```diff
@@ -22,4 +22,19 @@
     <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.1.1" />
   </ItemGroup>
 
+  <!--<ItemGroup>
```

Comment:
This is needed to make the `*.Designer.cs` auto generation. 
However, since the project targets both `net451` and `netstandard1.6`, the generated `Designer.cs` generates it for `net451`
i.e. 
```cs
typeof(MessageStrings).Assembly
```
instead of 
```cs
typeof(MessageStrings).GetTypeInfo().Assembly
```